src/scss/common/footer.scss

```diff
@@ -113,3 +119,126 @@
     }
   }
 }
+
+.footer-bottom-links a {
+  color: black;
+  padding-right: 30px;
+}
+
+.print-font {
+  font-family: $footer-bold-font;
+  color: black !important;
+}
+
+.footer-bottom-logos {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+}
+
+@media only screen and (min-width: 990px) {
+  .footer-bottom-logos img {
+    height: auto;
+    width: auto;
+  }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 989px) {
+  .links-col {
+    padding-top: 0.5rem !important;
+  }
+
+  .footer-bottom-info {
+    p {
+      padding-bottom: 1.5rem !important;
+    }
+  }
+
+  .footer-bottom-logos img {
+    height: auto;
+    width: auto;
+    max-width: 90%;
+    padding-left: 14px;
+  }
+}
+
+@media only screen and (min-width: 480px) and (max-width: 768px) {
+  .links-col {
+    padding-top: 0.5rem !important;
+  }
+
+  .footer-links .footer-support-links .h5, 
+  .footer-bottom-links .h5 {
+    margin-bottom: 15px;
+    text-align: left;
+  }
+  
+  .footer-links .footer-support-links a,
+  .footer-bottom-links a {
+    margin-bottom: 10px;
+  }
+
+  .footer-bottom-info {
+    p {
+      padding-bottom: 1.5rem !important;
+    }
+  }
+
+  .footer-bottom-logos img {
+    height: auto;
+    width: auto;
+    max-width: 90%;
+    padding-left: 14px;
+  }
+}
+
+@media only screen and (max-width: 480px) {
```

Comment:
I'd appreciate one more small pass over this new css to see what we can do in bootstrap. I'd like to get away from scss as much as possible to make our lives easier